src/ert/gui/simulation/view/progress_widget.py

```diff
@@ -110,5 +109,5 @@ def update_progress(self, status: dict[str, int], realization_count: int) -> Non
             self.stop_waiting_progress_bar()
         self.repaint_components()
 
-    def resizeEvent(self, a0: Any, event: Any = None) -> None:
+    def resizeEvent(self, event: Any = None) -> None:
```

Comment:
Is there a more specific event type we can use?

Reply:
there is. added it